README.md

```diff
@@ -1 +1,59 @@
-# test-bayes
+# Test Bayes
+
+[![Build Status](https://travis-ci.org/Test-Bayes/test-bayes.svg?branch=master)](https://travis-ci.org/Test-Bayes/test-bayes/)
+[![Coverage Status](https://coveralls.io/repos/github/Test-Bayes/test-bayes/badge.svg?branch=master)](https://coveralls.io/github/Test-Bayes/test-bayes?branch=master)
```

Comment:
Lose this. It isn't working and is broken anyways

Reply:
Done